app/scripts/lib/country-telephone-info.js

```diff
@@ -48,14 +48,33 @@
          return num.slice(0, 3) + ' ' + num.slice(3, 7) + ' ' + num.slice(7, 14);
        },
        normalize (num) {
-         if (/\+44/.test(num)) {
+         if (/^\+44/.test(num)) {
            return num;
          }
          return `+44${num}`;
        },
        pattern: /^(?:\+44)?\d{10,10}$/,
        prefix: '+44'
      },
+     RO: {
+       format(num) {
+         // +40 7xx xxxxxx
+         return num.slice(0, 3) + ' ' + num.slice(3, 6) + ' ' + num.slice(6, 12);
+       },
+       normalize(num) {
+         // allow +40 country code prefix
+         // as well as an extra 0 before the 7 prefix.
+         const prefix = /^(\+40)?0?/;
+         if (prefix.test(num)) {
+           num = num.replace(prefix, '');
+         }
+         return `+40${num}`;
+       },
+       // +407xxxxxxxx, allow leading 0 for sloppiness.
+       pattern: /^(?:\+40)?0?7\d{8,8}$/,
+       prefix: '+40'
```

Comment:
Should you prefill with `+407`, quick googling seems to imply that this there mobile number [prefix](https://www.howtocallabroad.com/romania/).